spec/requests/portfolio_items_spec.rb

```diff
@@ -362,14 +362,7 @@
         get "#{api}/portfolio_items/#{portfolio_item.id}/tags", :headers => default_headers
 
         expect(json["meta"]["count"]).to eq 1
-        expect(json["data"].first["name"]).to eq name
-      end
-
-      it "allows filtering on the response" do
```

Comment:
filtering no longer works for the `GET /tags` endpoint since it can only filter if there is a database column, and `:tag` is a generated column.

Reply:
@lindgrenj6 wouldn't the filtering on name and namespace work like this one
https://github.com/RedHatInsights/approval-api/blob/0ce80edc2ba633e6d8c0a8a2c74497e59cc8573f/app/services/mixins/tag_mixin.rb#L14

Reply:
No, not anymore since we're only returning the `:tag` field now. By removing `:namespace` and `:name` from the spec (since we don't want to return them) we can't pass them in the filter, heres the output after removing the name/namespace from the schema: `{"errors"=>[{"status"=>400, "detail"=>"Insights::API::Common::Filter::Error: found unpermitted parameter: name"}]}`

I'm not sure the proper way to handle this. I don't like that we're just returning the strings since that isn't an _actual_ column, it's due to that  we can't filter on it.